tests/transforms/test_mitigate.py

```diff
@@ -513,6 +523,15 @@ def test_polyfit(self):
         coeffs = qml.transforms.mitigate._polyfit(x, y, 2)
         assert qml.math.allclose(qml.math.squeeze(coeffs), [3, 2, 1])
 
+    def test_exponentialfit(self):
```

Comment:
Technically this is testing extrapolation rather than fitting (i.e. ignores B)

Reply:
True. Renamed in https://github.com/PennyLaneAI/pennylane/pull/5972/commits/52825e1353853d5d67a6481608b0810a284ea083.